File: API_R_01Agosto2024/Pedro/Q3.py
def main():
    NumeroCidades = int(input("Informe o número de cidades: "))

    mediaRenda = 0

    RendaPerCapita = float(input("\nInforme a Renda Per Cidade: "))

    contador = 1

    maiorRenda = RendaPerCapita
    menorRenda = RendaPerCapita

    somatorioRendas = RendaPerCapita

    if RendaPerCapita != 0:

        while contador != NumeroCidades:

            RendaPerCapita = float(input("\nInforme a Renda Per Cidade: "))

            contador += 1

            if RendaPerCapita > maiorRenda:
                maiorRenda = RendaPerCapita
            
            if RendaPerCapita < menorRenda:
                menorRenda = RendaPerCapita

            somatorioRendas += RendaPerCapita
            mediaRenda = somatorioRendas / NumeroCidades
    
    print(f"""
    Renda Media das Cidades -------> {mediaRenda}
    Maior renda             -------> {maiorRenda}
    Menor Renda             -------> {menorRenda}
""")
main()

File: API_R_01Agosto2024/Natiele Grazzielly/q3.js
import { getNumber, print } from "./funcoes.js"

function main(){

    const qtdCidades = getNumber('QTD DE CIDADES: ')

    let contador = 1
    let menorRenda = Infinity
    let maiorRenda = 0
    let somaRenda = 0

    while(contador <= qtdCidades ){

        let RendaInformada = getNumber("RENDA Per Capita da cidade " + contador + ": ")

        somaRenda += RendaInformada

        maiorRenda = RendaInformada
        if(maiorRenda < menorRenda){
            menorRenda = maiorRenda
        }

        contador++
    }

    let rendaMedia = somaRenda / qtdCidades

    const resultado = 
(`
=====================
RENDA MEDIA:${rendaMedia.toFixed(0)}
MENOR RENDA:${menorRenda}
MAIOR RENDA: ${maiorRenda}
=====================`)

    print(resultado)

}

main()